### PR TITLE
Added "parent type" to the lexicon

### PR DESCRIPTION
**Pull request by gribozavr**

### CODE REVIEW

**Comment by gribozavr** [comment]
@swift-ci Please test and merge

File: docs/Lexicon.rst [hunk 1]
```diff
@@ -291,6 +291,19 @@ source code, tests, and commit messages. See also the `LLVM lexicon`_.
     overlays" feature, which is used to augment a module with additional
     functionality when it is imported alongside another module.
 
+  parent type
+    The type where a given declaration is nested in. For example::
```
Comment (airspeedswift) [inline docs/Lexicon.rst]: "The type in which a given declaration is nested"

Comment (gribozavr) [reply docs/Lexicon.rst]: Thanks, fixed.

**Comment by gribozavr** [comment]
@swift-ci Please smoke test and merge.

**Comment by CodaFi** [comment]
Thanks!

@swift-ci please smoke test

**Comment by CodaFi** [comment]
@swift-ci please smoke test

**Comment by CodaFi** [comment]
⛵️

**Pull request merged**